vsce/vscode-extension/package.json

```diff
@@ -0,0 +1,210 @@
+        }
+      }
+    },
+    "menus": {
+      "view/title": [
+        {
+          "command": "rushstack.rushProjects.refreshEntry",
+          "when": "view == rushProjects",
+          "group": "navigation"
+        }
+      ],
+      "view/item/context": [
+        {
+          "command": "rushstack.rushProjects.revealInExplorer",
+          "when": "view == rushProjects && viewItem == project",
+          "group": "inline"
+        },
+        {
+          "command": "rushstack.rushProjects.runProjectScript",
+          "when": "view == rushProjects && viewItem == projectScript",
+          "group": "inline"
+        }
+      ]
+    },
+    "taskDefinitions": [
+      {
+        "type": "rush",
+        "required": [
+          "cwd",
+          "displayName",
+          "command",
+          "args"
+        ],
+        "properties": {
+          "cwd": {
+            "type": "string",
+            "description": "The working directory for the task"
+          },
+          "displayName": {
+            "type": "string",
+            "description": "The display name for the command"
+          },
+          "command": {
+            "type": "string",
+            "description": "The command to run"
+          },
+          "args": {
+            "type": "array",
+            "description": "The arguments to pass to the command"
+          }
+        }
+      },
+      {
+        "type": "rushx",
+        "required": [
+          "cwd",
+          "command"
+        ],
+        "properties": {
+          "cwd": {
+            "type": "string",
+            "description": "The working directory for the command"
+          },
+          "displayName": {
+            "type": "string",
+            "description": "The display name for the command"
+          },
+          "command": {
+            "type": "string",
+            "description": "The command to run"
+          }
+        }
+      }
+    ],
+    "views": {
+      "rushstack": [
+        {
+          "id": "rushProjects",
+          "name": "Projects"
+        },
+        {
+          "id": "rushCommands",
+          "name": "Rush Commands"
+        }
+      ]
+    },
+    "viewsContainers": {
+      "activitybar": [
+        {
+          "id": "rushstack",
+          "title": "RushStack",
+          "icon": "resources/rushstack.svg"
+        }
+      ]
+    },
+    "viewsWelcome": [
+      {
+        "view": "rushProjects",
+        "contents": "In order to use RushStack, you can open a folder containing a rush.json.\n[Open Folder](command:vscode.openFolder)\nTo learn more about how to use RushStack in VS Code [read our docs](https://rushstack.io).",
+        "when": "workbenchState == empty"
+      }
+    ]
+  },
+  "activationEvents": [
+    "onStartupFinished"
+  ],
+  "dependencies": {
+    "@microsoft/rush-lib": "workspace:*",
+    "@rushstack/node-core-library": "workspace:*",
+    "@rushstack/rush-sdk": "workspace:*",
```

Comment:
The trick to running webpack on `@rushstack/rush-sdk` is the `@rushstack/webpack-preserve-dynamic-require-plugin`, FYI.